.editorconfig

```diff
@@ -0,0 +1,10 @@
+# EditorConfig : http://EditorConfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
We could have the file get copied out as part of `npm i` or `start`, but that might be kinda wacky.